src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -156,6 +156,11 @@ private Runtime() {}
      * <p> The {@link System#exit(int) System.exit} method is the
      * conventional and convenient means of invoking this method.
      *
+     * @implNote
+     * If the {@link System#getLogger(String) the system logger} for {@code java.lang.Runtime}
+     * is enabled with logging level {@link System.Logger.Level#DEBUG Level.DEBUG} the stack trace
+     * of the call to {@code Runtime.exit()} is logged.
```

Comment:
Have you ruled out including the implNote in System.exit? Asking as I suspect it's more likely to be read there.

Reply:
I'll add an implNote to System.exit, and update the CSR.
```
     * @implNote
     * The initiation of the shutdown sequence is logged by {@link Runtime#exit(int)}.
```